Client/src/Pages/Monitors/index.css

```diff
@@ -74,10 +82,21 @@
 .incident-notif-config,
 .advanced-setting-config,
 .proxy-setting-config {
+  min-width: 350px;
   padding-right: 40px;
 }
 
 .monitors-create-button-holder {
   display: flex;
   justify-content: end;
 }
+
+#create-new-monitor-btn {
+  padding-top: 6px;
+}
+
+#ports-dropdown,
+#freq-dropdown {
+  width: 226.8px;
+  height: 20px;
+}
```

Comment:
get rid of hardcoded values

Reply:
There are some of the hardcoded values that we can not remove, since there are paddings and margins built-in for the MUI components and we need to set these hardcoded values according to that